SpatialGDK/Source/SpatialGDK/Public/LoadBalancing/AbstractLBStrategy.h

```diff
@@ -71,13 +72,15 @@ class SPATIALGDK_API UAbstractLBStrategy : public UObject
 	 * GetMinimumRequiredWorkers and SetVirtualWorkerIds are used to assign ranges of virtual worker IDs which will be managed by this
 	 * strategy. LastVirtualWorkerId - FirstVirtualWorkerId + 1  is guaranteed to be >= GetMinimumRequiredWorkers.
 	 */
-	virtual uint32 GetMinimumRequiredWorkers() const
-		PURE_VIRTUAL(UAbstractLBStrategy::GetMinimumRequiredWorkers,
-					 return 0;) virtual void SetVirtualWorkerIds(const VirtualWorkerId& FirstVirtualWorkerId,
-																 const VirtualWorkerId& LastVirtualWorkerId)
+	virtual uint32 GetMinimumRequiredWorkers() const PURE_VIRTUAL(UAbstractLBStrategy::GetMinimumRequiredWorkers, return 0;)
```

Comment:
Clang format really does a number on this class, is it worth replacing PURE_VIRTUAL with a similar implementation?
```{ 
    check(0); 
    return 0; 
}```

Reply:
should just need to put a semi colon at the end of the line (after the `PURE_VIRTUAL(...)`)